Exercises/chapter-11/e1-quiet-times.js

```diff
@@ -0,0 +1,23 @@
+async function activityTable(day) {
+    //Get an array of log file names for the specified day
+    const logFileList = await textFile("camera_logs.txt");
```

Comment:
logFileList and logFileArray are saying the same thing, but they're both wrong about what they contain. Can you try to find better names?